### ISSUE TITLE
I have problem when i start call 

### ISSUE THREAD

**Issue by fayyaddev** [EnableX/voice-api-outbound#2]
Hi 
Im using this app but i can't make call ! is there any problem in project ? please I want to help I have a graduated project.
![WhatsApp Image 2020-09-17 at 10 33 02 PM](https://user-images.githubusercontent.com/59565001/93518750-ed6abe00-f935-11ea-8a22-970b64f6a8ae.jpeg)

**Comment by rajeshgoyalg** [comment]
Hey @fayyaddev, I have just checked it and its working absolutely fine. Even, I tried by using your account and it worked. On your side, configurations looks good. Can you please try again and let me know if you still having the issue.

**Comment by rajeshgoyalg** [comment]
On further investigation, I found it is failing for your provided number 972592846625. I will get back to you. Thanks.

**Comment by fayyaddev** [comment]
Yes the number i have in my account domestic, but i try to call international.

**Comment by fayyaddev** [comment]
I wanna try know check >

**Comment by fayyaddev** [comment]
yes it's working, but  i want to make a international call . 
and i have this problem 
![error12](https://user-images.githubusercontent.com/59565001/93605976-222b5380-f9d0-11ea-9914-08f412dd9049.jpg)

**Comment by fayyaddev** [comment]
Also no voice message come just call without any voice   .

**Comment by rajeshgoyalg** [comment]
You can open the application on browser `http://localhost:3000` and it give you form to enter TTS text to play.
